.snyk

```diff
@@ -0,0 +1,6 @@
+exclude:
+  global:
+    - "vendor/**"
+    - "**/*_test.go"
+    - "pkg/controller/render/hash.go"
```

Comment:
For future reference and self-documentation, can we add one line comment on why we are excluding these go files.

Reply:
For posterity, Snyk provides a schema for adding things like reasons, expiration dates, etc. In practice, I couldn't get it to work correctly, so I just added comments to each line explaining the what and why.

Reply:
Looks great, thanks Zack!